ci/e2e-tests.sh

```diff
@@ -113,7 +121,12 @@ get_package() {
             printf '%s/%s\n' "$PWD" aziot-identity-service-*.x86_64.rpm
             ;;
 
-        'debian:9')
+        'platform:el8')
```

Comment:
Sort these arms alphabetically, in this and every other file.

Reply:
Done